core/tauri-runtime-wry/src/system_tray.rs

```diff
@@ -172,6 +176,17 @@ impl<T: UserEvent> TrayHandle for SystemTrayHandle<T> {
       .map_err(|_| Error::FailedToSendMessage)
   }
 
+  #[cfg(any(windows, target_os = "macos"))]
```

Comment:
No need for guard flags here I think, tao's `SystemTray::set_tooltip` exists on Linux but is a no-op.